src/types/app.ts

```diff
@@ -115,6 +115,11 @@ export type Currency = {
   minorUnits: number
 }
 
+export type Options = {
```

Comment:
`Options` is ambiguous
maybe can named `AppOptions`

Reply:
Okay! Thanks